seeds/connections-seed.js

```diff
@@ -0,0 +1,14 @@
+// Currently based of statuses, variants, should cover all possible scenarios within table boundries
+
+const company_name = 'Seeded company_name Name'
+const variants  = [{ status: 'pending', company_name }, { status: 'pending', company_name: '_@@$@*@&%£*&@£*$*(@£*$)@£*&' }, { status: 'unverified', company_name: 0 },  { status: 'unverified', company_name },{ status: 'verified_them', company_name }, { status: 'verified_us', company_name }, { status: 'verified_both', company_name }, { status: 'disconnected', company_name }]
```

Comment:
Looking at this we're only linting  the `src` directory. Can you update the prettier command in `package.json` to something like `prettier -c ./src ./migrations ./seeds ./test` and check it then formats this correctly?